### PR TITLE
Add springboot manifest

### PR DESCRIPTION
**Pull request by andreysaksonov**
There is a corresponding issue regarding "github" checkver.ps1 code: #2115

It is workarounded here by custom regex for now.

### CODE REVIEW

**Comment by rasa** [comment]
Does it make sense to have the word `.RELEASE` in the version?

File: bucket/springboot.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://projects.spring.io/spring-boot/",
+    "version": "2.0.0.RELEASE",
+    "license": "Apache 2.0",
+    "hash": "45362FCB133E5DB84A771DD37AF4D8B9655D936877F8E537DBDE2C1D648E488A",
```
Comment (brandon93s) [inline bucket/springboot.json]: The has is available at `$url.sha256`. Add the following to autoupdate to update the hash without needing to download the file: 

```json
"hash": {
    "mode": "extract",
    "url": "$url.sha256"
}
```

Comment (r15ch13) [reply bucket/springboot.json]: The `mode` property can be omitted, it defaults to `extract`

**Comment by andreysaksonov** [comment]
Replaced with PR to extras https://github.com/lukesampson/scoop-extras/pull/838

**Pull request closed without merging**